lib/chart/api_velocity_chart.es6.js

```diff
@@ -0,0 +1,159 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+let d3 = require('d3');
+
+function apiVelocityChartController($scope, $timeout) {
+  let createNewDiffView = function(minuend, subtrahend) {
```

Comment:
Comment added.
    // Create a view that contains set of APIs that is
    // minuend set minus subtrahend.